src/main/java/duke/MarkCommand.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+/**
+ * Encapsulate a command that allows user to mark existing Tasks as done,
+ * which is-a Command.
+ */
+public class MarkCommand extends Command {
+
+    private int taskNumber;
```

Comment:
```suggestion
    private final int taskNumber;
```
Suggesting the addition of `final` modifier.

Reply:
I did not implement it as final as I may need to modify the said variable later on.